Sources/MenuAnimator.swift

```diff
@@ -280,17 +307,27 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
         let animation : () -> Void = { [weak self] in self?.updateTransition(percentComplete: 1.0) }
         let completion : (Bool) -> Void = { [weak self] _ in
             if let transition = self {
-                let fromViewController = transition.transitionContext.viewController(forKey: UITransitionContextViewControllerKey.from)!
-                let toViewController = transition.transitionContext.viewController(forKey: UITransitionContextViewControllerKey.to)!
+                guard let transitionContext = transition.transitionContext else {
+                    fatalError("Invalid 'transition.transitionContext' value. This property should not be nil")
+                }
+                guard let contentSnapshotView = transition.contentSnapshotView else {
+                    fatalError("Invalid 'transition.contentSnapshotView' value. This property should not be nil")
+                }
+                let fromViewController = transitionContext.viewController(forKey: UITransitionContextViewControllerKey.from)!
+                let toViewController = transitionContext.viewController(forKey: UITransitionContextViewControllerKey.to)!
```

Comment:
Ok. I thought that it's ok in this case. User can easily find the reason of error.